ebu_tt_live/ui/user_input_producer/uip.js

```diff
@@ -508,14 +519,18 @@ $(document).ready(function() {
             */
             var media_offset = null;
             var scheduled_send_clock_type = $("#scheduled-send-clock-selector").val();
-            if (scheduled_send_clock_type == "media")
-            media_offset = scheduled_send_media_clock_offset;
-            var timeout = computeScheduledSendTimeout(media_offset);
-            var template_data = createTemplateDict();
-            var rendered_document = nunjucks.render('ebu_tt_live/ui/user_input_producer/template/user_input_producer_template.xml', template_data);
-            setTimeout(renderSendDocument, timeout, rendered_document);
-            notifySuccess($("#scheduled-confirmation-span"), "Scheduled...", true);
-            sequence_numbers[sequence_identifier] += 1;
+            if (scheduled_send_clock_type == "media") {
+              media_offset = scheduled_send_media_clock_offset;
+
```

Comment:
Previously the other statements that are now in the `{}` would have always executed regardless of the above `if`, now they are linked - I don't think this code tidy was supposed to change the functionality here?